test/system/src/main/java/io/pravega/test/system/framework/DockerRemoteSequential.java

```
@@ -0,0 +1,154 @@
+                log.error("Exception while copying test jar to the container ", e);
+                Assert.fail("Unable to copy test jar to the container.Test failure");
+            }
+
+            // Inspect container
+            final ContainerInfo info = CLIENT.inspectContainer(id);
+
+            // Start container
+            CLIENT.startContainer(id);
+
+        } catch (DockerException | InterruptedException e) {
+            log.error("exception in starting container ", e);
+            Assert.fail("Unable to start the container to invoke the test.Test failure");
+        }
+        return id;
+    }
+
+    private ContainerConfig setContainerConfig(String methodName, String className) {
+
+        Map<String, String> labels = new HashMap<>(1);
+        labels.put("testMethodName", methodName);
+        labels.put("testClassName", className);
+
+        HostConfig hostConfig = HostConfig.builder()
+                .portBindings(ImmutableMap.of( "2375/tcp", Arrays.asList( PortBinding.of( LoginClient.MESOS_MASTER, "2375" ) ) ) ).networkMode("host").build();
+
+        ContainerConfig containerConfig = ContainerConfig.builder()
+                .hostConfig(hostConfig)
+                .image(IMAGE)
+                .user("root")
+                .workingDir("/data")
+                .cmd("sh", "-c", "java -DmasterIP="+ LoginClient.MESOS_MASTER + " -DexecType="+getConfig("execType", "LOCAL")+ " -cp /data/build/libs/test-collection.jar io.pravega.test.system.SingleJUnitTestRunner " +
+                        className + "#" + methodName + " > "+  className + "#" + methodName + "server.log 2>&1" + "; exit $?")
+                .labels(labels)
+                .build();
+
+        return containerConfig;
+    }
+
+    private void stopContainer() {
+        try {
+            CLIENT.stopContainer(id, 0);
+        } catch (DockerException | InterruptedException e) {
+            log.error("Exception while stopping the container ", e);
```

This is logged at error level and suppressed.  Perhaps it should just throw and let the test fail.

---

done.